docs/develop/howtos/i18n.md

```diff
@@ -9,12 +9,35 @@ In order to translate strings, they have to be marked. This ensures that they wi
 ### Python
 
 !!! warning
-Do not use `f-strings` for translatable text. Interpolation is not supported for these types of strings (see [Issue](https://github.com/python-babel/babel/issues/594)).
+Do not use f‑strings for translatable text.
+F‑strings perform interpolation immediately, so the extracted message will contain the interpolated text (e.g. "Hello, max") rather than the intended placeholder (e.g. "Hello, {username}"). This means that your translators will see the wrong string and translations will not work as expected.
 
-Instead, use `format`:
-```diff
-- _(f"Hi {username}")
-+ _("Hi {username}".format(username=username))
+Similarly, do not pass keyword arguments to `_()` when using new‑style (curly‑brace) placeholders. For example, avoid this pattern:
+
+```python
+# ❌ Incorrect – the placeholder is not interpolated at runtime.
```

Comment:
indeed, the placeholder isn't replaced at runtime – because [`flask_babel.gettext()`](https://github.com/python-babel/flask-babel/blob/master/flask_babel/__init__.py#L628-L639) uses `s % variables` (old-style formatting), but for new-style formatting we would require `s.format(variables)`